app/components/publish-actions.tsx

```diff
@@ -149,9 +160,23 @@ const Embed = ({ chartWrapperRef, configKey, locale }: PublishActionProps) => {
                   "For embedding visualizations in systems without JavaScript support (e.g. WordPress).",
               })}
             />
+            <EmbedToggleSwitch
+              value="without-border"
+              checked={withoutBorder}
+              onChange={(_, checked) => handleStylingChange(checked)}
+              label={t({
+                id: "publication.embed.iframe.remove-border",
+                message: "Iframe without border",
```

Comment:
Nit: Maybe simply "Remove border" would suffice? I'd also move this to another line, as it's not about the same "category" as being responsive or not, but maybe @KerstinFaye has an opinion there 😅

Reply:
Yeah true, I first had is as radio button so then the text made more sense like so